client/src/components/Cell/CellMain.tsx

```diff
@@ -62,6 +75,19 @@ const CellMain = ({
     });
   }, [Alert, Confirm, brute, t]);
 
+  // Ascend
+  const ascend = useCallback(() => {
+    if (!brute) return;
+
+    Confirm.open(t('ascend'), t('ascendConfirm'), () => {
+      // TODO : Prompt for choice
+      Server.Brute.ascend(brute.name, 'regeneration').then(() => {
+        // Reload page
+        window.location.reload();
+      }).catch(catchError(Alert));
+    });
+  }, [Alert, Confirm, brute, t]);
```

Comment:
Instead of opening a dialog, clicking on the ascend button should redirect to a new `/ascend` route that displays a brute weapons/skills/pets like in the cell, and allow the user to click on any unlocked one to select it for ascension

Reply:
not done entirely, but I added the logic to display the ascension page and there's a start of display. so enough to resolve imo while still being wip
![image](https://github.com/user-attachments/assets/cb6f6b78-b38a-44ae-b545-5e89f674929c)